qa/L0_perf_nomodel/run_test.sh

```diff
@@ -25,15 +25,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
```

Comment:
Why remove this? It is in all our CI tests.

Reply:
I believe this check already happens in our test.sh file and so I believe it was no longer in run_test.sh file. Should we still have it in the run_test.sh as it is always run from the test file itself?

Reply:
You are right, keep it like it is.

Reply:
@deadeyegoodwin should I make this change across the other perf run_test files?

Reply:
I don't think they use the REPO_VERSION directly for this others... but you can check.